src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/NewAPIListWriter.java

```diff
@@ -83,17 +83,13 @@ protected void addContentSelectors(Content content) {
         if (releases.size() > 1) {
             Content tabs = HtmlTree.DIV(HtmlStyle.checkboxes,
                     contents.getContent("doclet.New_API_Checkbox_Label"));
-            for (int i = 0; i < releases.size(); i++) {
-                int releaseIndex = i + 1;
-                String release = releases.get(i);
-                HtmlId htmlId = HtmlId.of("release-" + releaseIndex);
-                tabs.add(Text.of(" ")).add(HtmlTree.LABEL(htmlId.name(),
-                                HtmlTree.INPUT(HtmlAttr.InputType.CHECKBOX, htmlId)
-                                        .put(HtmlAttr.CHECKED, "")
-                                        .put(HtmlAttr.ONCLICK,
-                                                "toggleGlobal(this, '" + releaseIndex + "', 3)"))
-                        .add(HtmlTree.SPAN(Text.of(release))));
+            // Table column ids are 1-based
```

Comment:
On line 82, which I cannot add a comment to on GitHub, there's this:

    List<String> releases = configuration.newAPIPageBuilder.releases;

It might be better to change it to this:

    List<String> releases = builder.releases;